src/components/headers/Header.js

```diff
@@ -54,7 +54,7 @@ const getComputedStyles = ({ align = 'center', as = 'h1' } :Props) => (
 
 const Header = styled.h1`
```

Comment:
Surely these have matching `<Typography />` variants available to help replace some of the style logic here. OFC there will still need to be some for media queries and stuff tho.